skillful_network_client/src/app/shared/components/user/profile-user-details/profile-user-details.component.ts

```diff
@@ -89,6 +89,15 @@ export class ProfileUserDetailsComponent implements OnInit {
     return response;
   }
 
+  cantBeFollow() : boolean{
+    let currentUser: User = this.authService.user;
+    let response = false;
+    if(currentUser.id == this.userPick.id){
+      response =  true;
+    }
+    return response;
```

Comment:
You can simplify by returning directly the value of the if condition somehow.
like this:
```ts
return currentUser.id == this.userPick.id;
```